builder/scaleway/step_snapshot.go

```diff
@@ -32,9 +32,16 @@ func (s *stepSnapshot) Run(ctx context.Context, state multistep.StateBag) multis
 		return multistep.ActionHalt
 	}
 
-	snapshot, err := instanceAPI.WaitForSnapshot(&instance.WaitForSnapshotRequest{
+	waitSnapshotRequest := &instance.WaitForSnapshotRequest{
 		SnapshotID: createSnapshotResp.Snapshot.ID,
-	})
+		Zone:       scw.Zone(c.Zone),
+	}
+	timeout := c.SnapshotTimeout
+	if timeout != 0 {
```

Comment:
We should have a default timeout explicitly stated here and eventually overridden. Could you initialize with the default timeout and eventually override it if it has been overridden by the user?